src/components/Pages/HomeSections/videoSection/VideoSection.tsx

```diff
@@ -1,36 +1,165 @@
 "use client";
 
-import React from "react";
+import React, { useEffect, useState } from "react";
 
-import "./videoSectionStyle.scss";
+import styles from "./videoSectionStyle.module.scss";
+import FreediLogoIcon from "@/components/icons/FreediLogoIcon";
+import FreediIcon from "@/components/icons/FreediIcon";
+import videoSectionImage from "@/assets/Images/VideoSectionImage.png";
+import videoSectionPeopleImage from "@/assets/Images/VideoSectionPeople.png";
+import Image from "next/image";
+import useWindowSizeFixed from "@/hooks/useWindowSizeFixed";
 
 export default function VideoSection({
     currentLang,
 }: Readonly<{
     currentLang: Record<string, string>;
 }>) {
-    const ref = React.useRef<HTMLIFrameElement>(null);
+    const { width } = useWindowSizeFixed();
+    const [isVideoPlaying, setIsVideoPlaying] = useState(false);
+    
+
+    const handlePlayVideo = () => {
+        setIsVideoPlaying(true);
+    };
+
+    if (width == undefined) {
+        return (
+            <section className={styles.videoSection}></section>
+        )
+    }
 
     return (
-        <section className="videoSection">
-            <h2 className="sectionTitle">
-                {currentLang["How does Freedi work"]}
-            </h2>
-            <p className="sectionText">
-                {
-                    currentLang[
-                        "Let us provide an explanation of how the application operates"
-                    ]
-                }
-            </p>
-            <iframe
-                ref={ref}
-                src="https://www.youtube.com/embed/qLznfpgj1E4"
-                allowFullScreen
-                title="Freedi"
-                className="videoSection__video"
-                loading="lazy"
-            />
+        <section className={styles.videoSection}>
+            {width! > 1200 ? (
+                <>
+                    <div
+                        className={styles.videoWrapper}
+                    >
+                        <iframe
+                            src={`https://www.youtube.com/embed/qLznfpgj1E4?wmode=transparent&autoplay=1`}
```

Comment:
Why does "How does FreeDi work?" and "Let us walk you through how our app works" not displayed with this condition?

Reply:
it is, look lines 68 - 76

Reply:
Those lines show different text. 
Take a look at lines 102-107.
They are missing for the large screen div.

Reply:
they are not missing, they are in different place in the dom in large screen. if you would run npm dev you can see it in large screen and under large screen.
on large screen they inside the video, so they are inside the video wrapper on react.
under large screen they like above the video, so they are first on react.

Reply:
Yes, you are correct. I missed it in the design of Figma.